homeassistant/components/deconz/strings.json

```diff
@@ -11,11 +11,14 @@
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
           "port": "[%key:common::config_flow::data::port%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of your deCONZ device."
```

Comment:
deCONZ is not its own device, its a service hosted on a machine

Reply:
right. Thank you for the catch. Would this work?